src/components/Card/index.story.js

```diff
@@ -0,0 +1,10 @@
+import { storiesOf } from '@storybook/react';
+import SetComponent from './SetComponent';
+
+const ImageBackground = '//evino-res.cloudinary.com/image/upload/v1615413100/web/assets/CuneBG.jpg';
+
```

Comment:
Acho que é válido colocar um exemplo aqui dele fluido também, com width 100%

Reply:
But the fluid option has a width of 100%